client/src/components/HandMarkedPaperBallot.tsx

```diff
@@ -315,30 +348,12 @@ const HandMarkedPaperBallot = ({
                 <Text>{date}</Text>
               </Prose>
             </PageFooterMain>
-            <PageFooterQRCode className="qr-code qr-code-page-1">
-              <QRCode
-                level="L"
-                value={ballotMetadata({
-                  isLiveMode,
-                  precinctId,
-                  ballotStyleId,
-                  pageNumber: 1,
-                  pageCount: 2,
-                })}
-              />
-            </PageFooterQRCode>
-            <PageFooterQRCode className="qr-code qr-code-page-2">
-              <QRCode
-                level="L"
-                value={ballotMetadata({
-                  isLiveMode,
-                  precinctId,
-                  ballotStyleId,
-                  pageNumber: 2,
-                  pageCount: 2,
-                })}
-              />
-            </PageFooterQRCode>
+            <PageFooterQRCode
+              className={qrCodeTargetClassName}
+              data-is-live-mode={isLiveMode}
```

Comment:
If `isLiveMode` is `false`, then the `dataset` will be `{ isLiveMode: "false" }`, and `!!"false"` is `true`. I think you'll need to fix that.

Reply:
fixed to `data-is-live-mode={isLiveMode === 'true}`